contributor_docs/access.md

```diff
@@ -0,0 +1,34 @@
+# Our Focus on Access
+
+At the [2019 Contributors Conference](https://p5js.org/community/contributors-conference-2019.html), we made a commitment to only add features to p5.js that increase access (meaning inclusion and/or accessibility). This means we prioritize the needs of marginalized groups over the continued comfort of more privileged groups with p5.js.
+
+This is not the first conversation we've had about access and inclusion. We share similar intentions in our [Community Statement](../CODE_OF_CONDUCT.md), which was written at the [2015 Contributors Conference](https://p5js.org/community/contributors-conference-2015.html).
+
+**Please consider this a starting point.** We want to invite more conversations about what access means and how we can prioritize it.
+
+## Kinds of access
+
+Increasing access is not a focus on expanding the raw number of people in the p5.js community. It is a focus on making p5.js available to and approachable for more people who are excluded from the p5.js community (intentionally or not) and people who are excluded from similar tools.
+
+Access here means making p5.js better for:
+
+- People who speak Spanish, Chinese, or other languages besides English
+- People who are Black, Indigenous, or people of Color
+- People who are lesbian, gay, bisexual, trans, or queer
+- People with marginalized genders
+- People who are disabled or sick
+- and other people who are systemically excluded
+
+### Examples
+
+These are examples of efforts we believe to increase access:
+
+- Translating more documentation and other materials into more languages
+- Improving out support for assistive devices (such as screenreaders)
+- Making p5.js error messages more helpful and supportive to developers
+
+There are other things we haven't thought of yet and we're excited to figure out what they are together. If you have an idea, please [share it as an issue](https://github.com/processing/p5.js/issues/new/choose).
+
+## Maintenance
+
+We also affirm our intention to maintain the existing feature set of p5.js. We'd like to fix bugs regardless of which area of the codebase they're in or how closely they are related to our focus on access.
```

Comment:
maybe replace "or how closely they are related to our focus on access." with something like "because we believe consistency of the tool makes it more accessible for beginners."